package.json

```diff
@@ -47,5 +47,10 @@
     "Firefox >= 54",
     "Edge >= 15",
     "ie >= 11"
-  ]
+  ],
+  "devDependencies": {
+    "@babel/helper-create-regexp-features-plugin": "^7.7.4",
+    "@babel/runtime": "^7.7.6",
+    "redux-mock-store": "^1.5.3"
```

Comment:
Added to get tests passing.

Reply:
Might not have needed the `@babel/`?

I was getting an error message that `@babel` couldn't load the regexp plugin which is why I tried to add it. 

 Now I can't figure out how to update the npm-offline-cache to get the build to pass. Tried
- `yarn install` hoping it would update everything automatically
- deleting `yarn.lock` and running `yarn install` but it didn't generate a new `yarn.lock` file
- deleting `node_modules/` and running `yarn install` but that didn't update the offline cache
- `yarn cache clean && yarn install --offline` succeeds on my local machine

Reply:
The trick here is the `.yarnrc` with turns on offline mode and pure lockfiles, which you need to disable when making an update (depending on how you make it ~_~)

Generally, `yarn cache clean && yarn add <some package>@pinnedVersion` should always work smoothly, but if there are ever problems, the full proof thing to do is to open the `.yarnrc` file and comment out the two flags in it, then `rm -rf node_modules && yarn cache clean && yarn add <some package>@pinnedVersion`, then reenable the `.yarnrc` flags

Reply:
💥 

Thanks for that info!!